src/MountedTraversal.js

```diff
@@ -50,6 +50,9 @@ export function instHasClassName(inst, className) {
     return false;
   }
   let classes = findDOMNode(inst).className || '';
+  if (typeof classes !== 'string') {
+    return false;
```

Comment:
The ```classList``` property is a ```DOMTokenList``` ([docs](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList)), so I think the truthiness test is okay?